collector/settings.mdx

```diff
@@ -409,13 +402,12 @@ Only relevant if you are running your database in Azure using Azure Database for
     <tr>
       <td>azure_ad_tenant_id (<code>AZURE_AD_TENANT_ID</code>)</td>
       <td>[none]</td>
-      <td>The "Directory (tenant) ID" on your application. Only necessary if not using the recommended Managed Identity setup; see these <a href="https://pganalyze.com/docs/log-insights/setup/azure-database#setting-up-authentication-using-azure-ad-application-alternative-to-managed-identity">setup instructions</a>
-      for details</td>
+      <td>The "Directory (tenant) ID" on your application. Only necessary if not using the recommended Managed Identity setup; see these <a href="https://pganalyze.com/docs/log-insights/setup/azure-database#setting-up-authentication-using-azure-ad-application-alternative-to-managed-identity">setup instructions</a> for details</td>
```

Comment:
Not new in this PR, but it feels slightly better to use "the" instead of "these" here to me:
```suggestion
      <td>The "Directory (tenant) ID" on your application. Only necessary if not using the recommended Managed Identity setup; see the <a href="https://pganalyze.com/docs/log-insights/setup/azure-database#setting-up-authentication-using-azure-ad-application-alternative-to-managed-identity">setup instructions</a> for details</td>
```